grunt/watch.js

```diff
@@ -1,60 +1,36 @@
+"use strict";
+
 module.exports = {
-    options:
-    {
-        atBegin: true,
```

Comment:
Only if we always use the `npm start` task, which per https://github.com/vokal/dominatr-grunt/pull/29, we don't.